components/report-details/support/transaction-status.tsx

```diff
@@ -29,7 +29,10 @@ const TransactionStatus = ({
 			<div
 				className={cn("flex justify-center", {
 					"animate-spin":
-						status.label !== "Confirmed" && status.label !== "Failed",
+						status.label !== "Confirmed" &&
+						status.label !== "Failed" &&
+						status.label !== "InsufficientFunds" &&
+						status.label !== "ActionRejected",
```

Comment:
```suggestion
						!(status.label === "Confirmed" ||
						status.label === "Failed" ||
						status.label === "InsufficientFunds" ||
						status.label === "ActionRejected"),
```
just I think it's cleaner...maybe :p